aries_cloudagent/config/argparse.py

```diff
@@ -680,7 +680,7 @@ def add_arguments(self, parser: ArgumentParser):
         parser.add_argument(
             "--monitor-revocation-notification",
             action="store_true",
-            env_var="ACAPY_NOTIFY_REVOCATION",
+            env_var="ACAPY_MONITOR_REVOCATION_NOTIF",
```

Comment:
The abbreviation feels slightly out of place but it's also one of the longest monitoring subjects so I'm not feeling particularly nit-picky on that point

Reply:
That's the main reason why I shortened it, because it's such a long environment variable

Reply:
https://developer.apple.com/documentation/coremedia/1489454-cmmetadataformatdescriptioncreat I mean, could be worse .... 🙃